readthedocs/oauth/services/github.py

```diff
@@ -36,6 +36,10 @@ class GitHubService(UserService):
     url_pattern = re.compile(r"github\.com")
     supports_build_status = True
 
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self._organizations_cache = {}
```

Comment:
If we're setting this on all of the classes, should it just be in the base model?